editor/src/core/es-modules/package-manager/utopia-api-typings.ts

```diff
@@ -1,12 +1,5 @@
 export const utopiaApiTypings = `declare module 'utopia-api/helpers/helper-functions' {
   import { PropertyControls } from 'utopia-api/property-controls/property-controls';
-  export function registerComponent(paramsObj: {
```

Comment:
Merging with master should remove this now - I missed this change previously but have since committed it